Source/WSTagsField.swift

```diff
@@ -109,6 +109,15 @@ open class WSTagsField: UIView {
             textField.keyboardType = newValue
         }
     }
+    
+    public var returnKeyType: UIReturnKeyType {
```

Comment:
Regarding issue #28, maybe we should turn the `textField` public. Users are requesting a way to customise the `textFiled`.